tests/mockers.py

```diff
@@ -484,3 +484,43 @@ def mock_derive_current_user_login(domain: str) -> str:
 def mock_input(msg: str) -> str:
     print(msg)
     return '1'
+
+
+def mock_is_file_true(file: Any) -> bool:
+    return True
+
+
+def mock_is_file_false(file: Any) -> bool:
+    return False
+
+
+def mock_is_file_not_github_token(file: Any):
+    if '.github-token' not in file:
+        return True
+
+
+def mock_os_environ_get_none(self, key: str, default=None) -> Optional[str]:
+    if key == GitHubToken.GITHUB_TOKEN_ENV_VAR:
```

Comment:
I wouldn't hardcode a behavior any specific env var, better make it configurable